javascripts/Start.js

```diff
@@ -3,12 +3,43 @@
 let start = false;
 let end = false;
 let pause = false;
+let request = require('request');
+let abc = "";
+abc.NOME = "Gabriel";
+const xpto = {};
+
+for (let i = 0; i < xpto.length; i++) {
+  xpto[i] = i;
+  console.log(xpto);
+}
+
+const array = [1, 2, 3];
+const b = array[5];
+
+async function teste() {
+  const res = await Promise
+    .resolve(foo())
+    .catch(err => {
+      console.log('Erro interno', err);
+      throw new Error('Erro 2');
+    });
+  return res;
+}
+
+function foo() {
+  return Promise.reject('Erro 1');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/gabholanda/module01-game/pullRequest?prid=8950836)
```suggestion
  return Promise.reject("Erro 1");
```